### DISCUSSION TITLE
Generating paf files

### DISCUSSION THREAD

**Discussion by MehdiiYac** [GMOD/jbrowse-components#3587] [Q&A] [answered]
Does the order matter when creating .paf files 
for example does
• minimap2 file1.fa file2.fa > file_1vs2.paf
• minimap2 file2.fa file1.fa > file_1vs2.paf
generate the same file.
Thanks

**Comment by cmdcolin** [comment, marked as answer]
they are not equivalent, the first argument is the target and the second file is the query. it will affect output. generally you can consider aligning a "draft" as query vs a "reference" as target as a good strategy, but if both are reference quality it may be more arbitrary. you can also synthesize "reciprocal best hits" which would essentially combine the output of both commands, or use another method. I cannot advise in detail how to do whole genome alignments, and note that minimap2 is only one of the formats we support (we also can natively import mummer, liftover chain files, mashmap, and mcscan data as synteny tracks)

**Reply by MehdiiYac** [reply]
Thanks !
